### DISCUSSION TITLE
units for groundwater related variables

### DISCUSSION THREAD

**Discussion by abbylute** [iiasa/CWatM#91] [Q&A] [answered]
Hello, I am trying to compare daily baseflow data with daily discharge data. I see in the list of outputs (https://cwatm.iiasa.ac.at/listVariables.html) that baseflow has units of m whereas discharge has units of m3/s. I also saw a variable MtoM3. My initial thought is that I should multiply the daily baseflow data (m) by the conversion factor MtoM3 to get baseflow in units of m3/day, and then I could convert to m3/s to match the discharge data. I have done this and the resulting values are smaller than I expected. Is this the correct approach?

Thanks!

**Comment by dof1985** [comment]
Hi @abbylute,

Thanks for approaching. I think your approach is correct.  You can also multiply by cellArea_totalend to convert M to M3. For the discharge, you can multiply by the number of seconds in a day (i.e., 86400) to change from M3/s to M3, or divide the baseflow by 86400 to have it as M3/s (as you have probably done). 

The main issue is that baseflow in CWatM is only part of the river flow (e.g., as shown in discharge). Baseflow is the water flow into river from groundwater sources. Other sources ti river in the model are direct surface runoff, interflow from soil, return flows, etc.

I hope this makes sense.

Dor

**Comment by abbylute** [comment]
Thanks Dor for your help. That is good to know that I am doing it correctly.

To clarify, is baseflow routed along the river network like discharge is? If I output a discharge timeseries (e.g. discharge_daily.tss) at a given point along the river I get the discharge of the river at that point. If I output the same for baseflow. (e.g. baseflow_daily.tss), will I get something similar (i.e. the baseflow component of the river discharge at that point) or will I only get the baseflow from that grid cell?

Thanks for your help!

**Reply by dof1985** [reply]
Dear @abbylute 

To start, every grid cell in CWatM simulates a channel. Some channels are very low order (e.g., not necessarily channels you see in vectorized river networks); others would be main, high-order channels. At the same time, the model simulates processes occurring on the surfaces outside of the channel (e.g., runoff), soil and sub-surface (percolation, groundwater recharge), and the interfaces between these compartments - interflow, baseflow, etc.

So - discharge characterizes the channel, and at a point over time, you can look at the discharge graph within the channel. Baseflow in the model, however, characterizes the interface between groundwater and channels - namely, the water captured by channels from the groundwater compartment at a given time step. These water are input into channels and are part of the discharge.

Hope this make more sense now.

Dor

**Reply by abbylute** [reply]
Hi Dor,

Thank you for your explanation, that is very helpful in conceptualizing how the model works. So it sounds like baseflow values will all be local values, not aggregated across some basin or something. Would there be any way then to get at the accumulated baseflow at a point on the river reach? Something like the plot below? I was hoping to understand the contribution of groundwater to stream discharge.

![image](https://github.com/iiasa/CWatM/assets/30153868/5c2937cd-2f5a-4339-8d5b-8e95d2fa9618)

Thanks,
Abby

**Reply by dof1985** [reply]
Hi @abbylute 

I want to make sure that we don't mix it up - by baseflow you mean water flow in a channel that originates from groundwater, correct? or do you mean the long-term low-flow conditions, e.g., environmental flow?

If the first holds true you can use zonal statistics with a catchment mask to sum all river inputs and to plot them as you suggest.
For a daily output, it would look like this (more or less):
returnflowNonIrr_daily + returnflowIrr_daily + sum_directRunoff_daily + baseflow_daily + lakeResOutflowM

To get the discharge correctly you should also add inflows from upstream and outflows downstream (e.g., using the discharge variable), inflows into reservoirs (outflows from the river), abstraction from the channels, and evaporation losses.

Best

Dor

**Reply by abbylute** [reply]
Hi Dor,

Yes, by baseflow I mean the flow to the channel that originates from groundwater. 

I don't quite understand your suggestion. Is your suggestion to sum up the grid cell values of the variables you listed in order to get an aggregate time series of discharge? Then I could compare that with a spatially aggregated value of baseflow? Is this an alternative to using the routed discharge that the model provides so that it can be more fairly compared with the baseflow that does not get routed? 

Thank you again for your help, this is very helpful in understanding how the model works and how to interpret the outputs.

Best,
Abby

**Reply by dof1985** [reply, marked as answer]
Hi @abbylute 

I would advise you look in the model .py files to understand the processes better. I would bring some examples from there to give you some hints for how it works.

1. Routing is done in routing_kinemtaic.py: what is routed is water in channels, lakes, and reservoirs. After entering the channel, the water is not divided by source, rather, it is all treated as channel water.

2. Channel water are affected by multiple inflows and out flows; one of which is baseflow. You can review the runoff_concentration.py to find the following:
self.var.runoff = self.var.sum_landSurfaceRunoff + self.var.baseflow + self.var.leakageIntoRunoff

Whereas self.var.runoff is the variable fed into channels later on; self.var.sum_landSurfaceRunoff is both direct runoff and interflow, self.var.leakageIntoRunoff comes from Canals and self.var.baseflow is baseflow.

3. Finally, since baseflow is not routed on its own, but with all other water sources within the channel, you can only analyze it at a catchment or sub-catchment cell by calculating the relative share of baseflow from all inflows into the river per time step.

Good luck
Dor

**Reply by abbylute** [reply]
Thank you Dor, this is very helpful!
